CONTRIBUTING.md

```diff
@@ -0,0 +1,137 @@
+# Contributing Guidelines for ITCR Course Template
+
+This template includes all of the files that you need to get started creating your course in [R Markdown](https://rmarkdown.rstudio.com/) using the [bookdown package](https://bookdown.org/).
+This is useful as this package allows you to preview your course easily from RStudio and to eventually host a book on GitHub and Leanpub.
+
+## Creating your course
+
+In the upper right of this screen, click `Use this template` and follow the steps to set up your course's GitHub repository.
+
+TODO: Add more details here about setting up repositories.
+
+## Setting up Docker image
+
+Particularly for courses that involve running example code, it's recommended that you use a Docker image for development to maintain software version consistency across course developers.
+If you haven't installed Docker, you can do [so here](https://docs.docker.com/get-docker/).
+You will need to create a Docker account if you don't have one.
+
+To pull the docker image associated with this template, you can run this command in your command line.
+
+```
+docker pull jhudsl/itcr_course_template
+```
+
+To use the Docker image associated with the course template, first navigate to the the top of this GitHub repository.
+Now you can start up the Docker container using the command below.
+Replace `password` with something else of your choosing
+
+```
+docker run -it -v $PWD:/home/rstudio -e PASSWORD=password -p 8787:8787 jhudsl/itcr_course_template
+```
+
+Now, if you run the command: `docker ps`, you should see a container is up and running!
+
+In a web browser navigate to the following to get to an RStudio that is ran from your Docker container.
+```
+localhost:8787
+```
+
+You'll need to use `rstudio` as the username and whatever password you chose to log in.
+
+### Starting a new Docker image
+
+TODO: Add more information about how to start a new image and add packages to existing ones.
+
+## Citations
+
+TODO: How to add and use citations
+
+## Spell check
+
+TODO: Can have github actions run this for us.
+
+## Style guide
+
+TODO: Are there any standards we wanna keep to
+
+## Bookdown Rendering
+
+As you modify the names of the chapters of your course and add more chapters (using the `.Rmd` files), you need to update the `_bookdown.yml` file accordingly.
```

Comment:
This material is directly copied over from the content that @carriewright11 wrote in the original README. Github hasn't recognized that though.